modules/carto/src/api/basemap.ts

```diff
@@ -0,0 +1,75 @@
+import {Basemap, KeplerMapConfig} from './types';
+
+const getRasterJsonMapStyle = (basemap: Basemap) => ({
+  version: 8,
+  sources: {
+    'basemap-tile-source': {
+      type: 'raster',
+      tiles: [basemap.settings.url],
+      tileSize: 256
```

Comment:
256 seems like a pretty small tile size, is that correct?

Reply:
This is some magic number that we agreed upon and that works well for most sources we tested.

Anyway, i think we should make it confiruable.
I'll try to make it part of protocol, so it's already part of `keplerMapConfig` and we don't have to hardcode it here.

Good point.

Reply:
Finally, i've decided that it's better to keep whole style, so app that generates `keplerMapConfig` has complete control over what style is in basemap (as side-effect, we don't need code that generates style in this module).